libs/dsb-client-gateway-storage/src/lib/module/acks/entity/pending-acks.entity.ts

```diff
@@ -14,6 +13,9 @@ export class PendingAcksEntity {
   @PrimaryColumn()
   public clientId: string;
 
+  @PrimaryColumn()
```

Comment:
Is this supposed to form part of the primary key?

Reply:
for case search with date are created new clientid in nats not shared with existing clientid